Onboarding/onboarding-schedule-template.md

```diff
@@ -0,0 +1,43 @@
+# Ondboarding schedule template
+
+## Post onboarding (1 week before)
+- Monday
+    - Send welcome email (this should include the intake questionaire and things they should expect going forward)
+    - Create a place to store all relavent documents for the team we are supporting
+    - Send invatations for onboarding meetings
+    - Send list of contact information about the support team
+- Tuesday
+    - Setup Slack accounts and invites
+    - Send instruction for setting up GitHub
+- Wednesday
+    - Connect GitHub account to any repos that are needed
+    - Setup ZenHub for team
+
+## Onboarding week
+- Monday
+    - Onboarding meeting/Kickoff meeting
+- Tuesday
+    - Engineering meeting 
+    - Design meeting
+    - Product meeting 
+- Wednesday 
+    - Meeting to make sure the team is setup and ready for development
+    - Strategy and implementation meeting (discuss how we might integrate the project)
+-  Thursday
+    - Start support
+    - Schedule times to meet with team
+
+## Post onboarding
+Post onboarding should be determined by project and team experience.
```

Comment:
Do you have a sense what experience(s) we should be looking for with regard to post onboarding assistance? Are there things we can point to as predictors of success or that have worked well for us in the past?

Reply:
We only have what Jeff has shared with us. I don't have hand on experience with external teams so all we have really is Jeff and Nicks experience with external teams. He said it really depends on the experience with the team. Some don't need too much help and some do. I think we just play it by ear for now and just figure it out as we go.